hound/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hound
+  labels:
+    app: hound
+spec:
+  selector:
+    app: hound
+  type: LoadBalancer
+  # FIXME(dims) - we need a fixed IP address for the load balancer
+  #loadBalancerIP: 104.197.177.166
+  ports:
```

Comment:
we really need to get kube-lego or something running in the utility cluster, for HTTPS

Reply:
Ack.

Reply:
indeed. https://github.com/kubernetes/k8s.io/issues/38 :)